docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,335 @@
+# Window tutorial
+
+This tutorial describes how to create and manage windows in NUI, covering the following subjects:
+
+[Overview](#overview)<br>
+[Creating Window](#creating)<br>
+[Deleting Window](#deleting)<br>
+[Resizing Window](#resizing)<br>
+[Moving Window](#moving)<br>
+[Showing & Hiding Window](#showinghiding)<br>
+[Changing Window Stacking Order](#raisinglowering)<br>
+[Event Handling](#eventhandling)<br>
+[Window Rotation](#rotation)<br>
+[Multi Window](#multiwindow)<br>
+
+<a name="overview"></a>
+## Window overview
+
+A window contains the visible content of the application.
+
+When an application is created, it automatically creates the [Creating Window](#defaultwindow) for building the main user interface of the application.
+
+[Additional windows](#additionalwindow) can be created (as needed) to display additional content.
+
+The window delivers [various events](#eventhandling) (e.g. key event and touch event) to the application.
+
+<a name="defaultwindow"></a>
+## Creating window
+
+When an application is created, the default window will be automatically created:
+ ```csharp
+ MyApplication myApp = new MyApplication();
+ ```
+
+By default, the default window is created in full screen size.
+
+It also allows to specify the initial size and position of the default window by passing them to the constructor of the application:
```

Comment:
Suggest rephrasing to:

You can specify the initial size and position of the default window using the constructor of the application.

Please verify.